Formula/insighttoolkit.rb

```diff
@@ -0,0 +1,111 @@
+class Insighttoolkit < Formula
+  desc "ITK is a toolkit for performing registration and segmentation"
+  homepage "https://www.itk.org"
+  url "https://downloads.sourceforge.net/project/itk/itk/4.12/InsightToolkit-4.12.2.tar.gz"
+  sha256 "8ba2183111723c4405af80724c60d07484c2bff8120ee4cef543bb726d07855e"
+  head "git://itk.org/ITK.git"
+
+  option "without-python", "Build without python2 support"
+
+  depends_on "cmake" => :build
+  depends_on :python3 => :optional
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on "hdf5"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "gdcm"
```

Comment:
non-build deps should be alphabetized amongst themselves

Reply:
Done